public_api.ts

```diff
@@ -1 +1,9 @@
-export * from './src/app/ng-intercom'
+export { Intercom } from './src/app/ng-intercom/intercom/intercom'
+export { IntercomConfig } from './src/app/ng-intercom/shared/intercom-config'
+export { IntercomHideDirective } from './src/app/ng-intercom/directives/hide.directive'
+export { IntercomModule } from './src/app/ng-intercom/intercom.module'
+export { IntercomShowDirective } from './src/app/ng-intercom/directives/show.directive'
+export { IntercomShowMessagesDirective } from './src/app/ng-intercom/directives/show-messages.directive'
+export { IntercomShowNewMessageDirective } from './src/app/ng-intercom/directives/show-new-message.directive'
+export { IntercomShutdownDirective } from './src/app/ng-intercom/directives/shutdown.directive'
+export { IntercomTrackEventDirective } from './src/app/ng-intercom/directives/track-event.directive'
```

Comment:
I think this should be enough, but I don't want to take a risk, so let me know if you want me to update it : 
```ts
export { Intercom } from './src/app/ng-intercom/intercom/intercom'
export { IntercomConfig } from './src/app/ng-intercom/shared/intercom-config'
// export { IntercomHideDirective } from './src/app/ng-intercom/directives/hide.directive'
export { IntercomModule } from './src/app/ng-intercom/intercom.module'
// export { IntercomShowDirective } from './src/app/ng-intercom/directives/show.directive'
// export { IntercomShowMessagesDirective } from './src/app/ng-intercom/directives/show-messages.directive'
// export { IntercomShowNewMessageDirective } from './src/app/ng-intercom/directives/show-new-message.directive'
// export { IntercomShutdownDirective } from './src/app/ng-intercom/directives/shutdown.directive'
// export { IntercomTrackEventDirective } from './src/app/ng-intercom/directives/track-event.directive'
```